shapely/ops.py

```diff
@@ -319,29 +319,12 @@ def nearest_points(g1, g2):
 
 
 def snap(g1, g2, tolerance):
-    """Snap one geometry to another with a given tolerance
-
-    Vertices of the first geometry are snapped to vertices of the second
-    geometry. The resulting snapped geometry is returned. The input geometries
-    are not modified.
-
-    Parameters
-    ----------
-    g1 : geometry
-        The first geometry
-    g2 : geometry
-        The second geometry
-    tolerance : float
-        The snapping tolerance
+    """
+    Snaps an input geometry (g1) to reference (g2) geometry's vertices.
```

Comment:
Suggest changing the parameters to `geometry`, `reference` to match the underlying function instead of `g1` and g2`.

Should probably still retain the parameters part of the docstring, but the more verbose description and examples can be dropped in favor of `shapely.snap`.

Reply:
Changing parameter names would be a breaking change for users who used kwargs.

Reply:
These are required positional parameters; hopefully the risk of changing the parameter names is very low in this case?  i.e., it would be weird to call it like

```
shapely.ops.snap(tolerance=12, g2=Point(0, 1), g1=Point(10, 10)
```


I agree that changing optional named parameters is a breaking change; I'm not sure if we are as strict about simpler cases of required positional parameters (can't remember a recent precedent either way)

Reply:
@brendan-ward I'm marking this as resolved and will open a follow-up issue to discuss parameter renaming as I feel this is out of scope of this PR.